lib/queue.js

```diff
@@ -223,6 +223,91 @@ Queue.prototype.checkHealth = function (cb) {
   return promise;
 };
 
+Queue.prototype._scanForJobs = function(cursor, set, cb) {
+  // Ideally, we would be able to paginate redis sets like we do sorted ranges.
+  // For an MVP, however, we're aiming to replace our usage of Arena for Bull 1.x
```

Comment:
add a ',' - I couldn't figure out that the end of this line was the end of a clause